src/client/visualization.tsx

```diff
@@ -54,11 +54,28 @@ function getChartType(
     /**
     [1, "Visualization", "{"Visualization":"piechart","Title":null,"XColumn"…"]
     */
-    if (queryPropertiesTable._rows[0][1] !== 'Visualization') {
+    if (
+        queryPropertiesTable._rows[0][1] !== 'Visualization' &&
+        // This is how we get Visualization for AppInsights.
+        !(queryPropertiesTable._rows[0][0] as string).includes('Visualization')
```

Comment:
The AppInsights endpoint doesn't return the same schema.

Reply:
npm package doesn't seem to support AppInsights endpoints or I'm doing something wrong.